### PR TITLE
Connect to bookstore API using axios to fetch, add and remove book

### PR DESCRIPTION
**Pull request by jecode93**
In this milestone I: 

- Use the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) as backend.
- Use [`axios`](https://axios-http.com/docs/intro) with [`createAsyncThunk`](https://redux-toolkit.js.org/api/createAsyncThunk) to make HTTP requests to the backend
- Fetch my `books` from the Bookstore API **after** the component has rendered
- Refactor your **add book** and **remove book** features to persist changes in the server

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import booksReducer from './books/booksSlice';
 
 const store = configureStore({
   reducer: {
-    book: booksReducer,
+    books: booksReducer,
     // categories: categoriesReducer,
   },
 });
```
Comment (Munsa1) [inline src/redux/store.js]: - [ ] You have commented code in your codebase in this file, remove it for a more cleaner codebase

File: src/components/Books.jsx [hunk 2]
```diff
@@ -1,27 +1,37 @@
...
+  const { isLoading } = useSelector((state) => state.books.isLoading);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBook());
+  }, [books, dispatch]);
+
+  if (isLoading) return <div>Loading...</div>;
+  // if (error) return <div>Error</div>;
```
Comment (Munsa1) [inline src/components/Books.jsx]: - [ ] Again here you have commented code in your codebase in this file, remove it for a cleaner codebase

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @jecode93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
App Works As Expected :heavy_check_mark: 
Student Has Met The Technical Projects Requirements :heavy_check_mark: 
Styling Is Nice :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Due to insufficient files changed I will leave a comment here

- [ ] Please remove the link to the `Live Demo` section from the table of contents as it links to a section that has been removed making it broken.
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
Hi @jecode93 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work using Axios 🥇 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**